src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -541,25 +542,12 @@ public function testSerializerCacheDisabled()
      */
     public function testDeprecatedSerializerCacheOption()
     {
-        $deprecations = array();
-        set_error_handler(function ($type, $msg) use (&$deprecations) {
-            if (E_USER_DEPRECATED !== $type) {
-                restore_error_handler();
+        ErrorAssert::assertDeprecationsAreTriggered('The "framework.serializer.cache" option is deprecated', function () {
```

Comment:
I would send another PR when this is merged for the `2.7` branch. Or would you prefer if we reverted the changes to the tests before merging and starting with them on `2.7`?